src/water.rs

```diff
@@ -136,17 +136,19 @@ fn setup_water(
   commands
     .spawn((WaterTiles, Name::new("Water")))
     .with_children(|parent| {
-      let grid_center = (WATER_SIZE * WATER_GRID_SIZE) as f32 / 2.0;
```

Comment:
If using a non-default grid size via "spawn_tiles" in settings, the grid would not be centered, since the centering assumed 6 tiles in each direction. Therefore proposing to read tile amount from grid.x and grid.y instead.